mobile/src/main/java/ch/epfl/sdp/mobile/ui/speech_recognition/ChessSpeechRecognitionFilter.kt

```diff
@@ -0,0 +1,210 @@
+package ch.epfl.sdp.mobile.ui.speech_recognition
+
+import java.lang.StringBuilder
```

Comment:
You don't need to import `StringBuilder`. Kotlin has this already integrated

Reply:
Good to know !